### ISSUE TITLE
Financial Market News Sentiment Analysis

### ISSUE THREAD

**Issue by puppalamythrireddy3** [puppalamythrireddy3/Financial-project#1]
Sentiment-Analysis-for-Predicting-Stock-Market-Movements-using-News-Headlines
Sentiment Analysis for Predicting Stock Market Movements using News Headlines

Description
Sentiment analysis can be used to analyze news headlines in order to forecast the movement of the stock market. The rationale behind this method is that news headlines can contain information that might affect the mood of investors, and therefore influence stock prices. Trying to predict the stock market movements is always a hard task, most of the people fail at this challenge, many studies have shown that news sentiment can be a valuable predictor and this has led to to the development of various sentiment analysis techniques and tools for financial analysts and investors. By analyzing the sentiment of news headlines related to specific companies or sectors, investors can gain insights into the potential movements of stock prices and make informed investment decisions. The main objective of this project is to explore the use of sentiment analysis techniques to predict stock market movements by analyzing news headlines. The goal is to identify the sentiment of the headlines of the day and extract meaningful insights that can inform investment decisions. The aim is to develop a reliable and accurate predictive model that can be used to support investment strategies and maximize returns in the stock market. The project was entirely developed in Python and Jupyter notebooks. We have analyzed the dataset to understand the distribution of the labels and how they are related to each other. Noise has been removed from each headline also the stopwords, reducing the remaining terms to the basic form (stemming and lemmatization). We have apply varius classifiers, such as random forest,XGBoost,and more advanced models like Convolutional Neural Network, Long Short-Term Memory(LSTM) and pre-trained models like BERT and RoBERTa. For more details and information about the dataset, please refer to Kaggle’s page
